tests/fixtures/fixtures_logger.py

```diff
@@ -261,9 +261,9 @@ def verkehrsleistung_time_df():
         {
             "time": [
                 datetime(2020, 1, 1, 0, 1 if i == 6 else 0, (10 * i) % 60)
-                for i in range(1, 7)
+                for i in range(0, 7)
```

Comment:
simply `range(7)` works too :)

Reply:
Yes, you are right. I would just leave it now like this, because this makes it more explicit